tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -0,0 +1,56 @@
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: For openSUSE virtualization test only. login in console, install kvm/xen patterns if needed.
+#          Even if you'd like to run tests without host installation(IPMI_DO_NOT_RESTART_HOST=1), this module is still necessary as login console in this module is required.
```

Comment:
I think you need more introduction and comment to this new module. Its purpose, functionality, why it is needed and how it can be called or used.

Reply:
module renamed and comments added.